plugins/org.jboss.reddeer.swt/src/org/jboss/reddeer/swt/impl/text/LabeledText.java

```diff
@@ -37,6 +38,6 @@ public LabeledText(String label) {
 	 * @param label the label
 	 */
 	public LabeledText(ReferencedComposite referencedComposite, String label) {
-		super(referencedComposite, 0, new WithLabelMatcher(label));
+		super(referencedComposite, 0, new WithLabelMatcher(new RegexMatcher(label + "\\s*\\**")));
```

Comment:
I wouldnt use regex matcher inside this in this way. Because if a label contains a special char... it would not be matched properly.